### PR TITLE
Fix/accessibility

### PR DESCRIPTION
**Pull request by elsafluss**
### Is this a fix or a feature?
fix
### What is the change/fix?
Adjustments to CSS and HTML for accessibility
### Where should the reviewer start?
styles.css
### How should this be tested?
open the site, run lighthouse

### CODE REVIEW

File: src/_two-cards.scss [hunk 1]
```diff
@@ -18,45 +18,45 @@
...
         background-color: $blue-green;
         border-radius: 5px;
         box-shadow: 2px 2px 10px #37515C;
+        font-size: 100%;
         width: auto;
 
         &:hover {
-            cursor: pointer;
             background-color: darken($blue-green, 20%);
+            cursor: pointer;
```
Comment (colefiscus) [inline src/_two-cards.scss:59]: A nice nested rule!

File: src/recipe.js [hunk 2]
```diff
@@ -1,26 +1,15 @@
...
-        if (specificIngredient.id === ingredient.id) {
-          costCounter += (Number(specificIngredient.estimatedCostInCents) *
-          Number(ingredient.quantity.amount))
-        }
-      })
-    });
-    return costCounter;
+class Recipe extends User {
+  constructor(recipe, ingredientsData, id, name, pantry) {
+    super(id, name, pantry, ingredientsData)
```
Comment (colefiscus) [inline src/recipe.js:5]: I'm not entirely sure this is right, but I don't know enough to know that it's definitely wrong..

File: src/recipe.js [hunk 3]
```diff
@@ -1,26 +1,15 @@
...
-    this.ingredients.forEach(ingredient => {
-      this.ingredientsData.find(specificIngredient => {
-        if (specificIngredient.id === ingredient.id) {
-          costCounter += (Number(specificIngredient.estimatedCostInCents) *
-          Number(ingredient.quantity.amount))
-        }
-      })
-    });
-    return costCounter;
+class Recipe extends User {
```
Comment (DrewBradley) [inline src/recipe.js:3]: Way to go with the inheritance!

File: src/scripts.js [hunk 4]
```diff
@@ -157,14 +156,13 @@ function displayDirections(event) {
       getIngredients()
         .then((ingredientsData) => {
           let recipeObject = new Recipe(recipe, ingredientsData)
-          let cost = recipeObject.calculateCost()
-          let costInDollars = (cost / 100).toFixed(2)
+          let cost = recipeObject.calculateCost(recipeObject.ingredients, recipeObject.ingredientsData)
```
Comment (colefiscus) [inline src/scripts.js:159]: Glad we figured this out!!

File: src/scripts.js [hunk 5]
```diff
@@ -189,15 +187,15 @@ function displayDirections(event) {
               ${ingredient.quantity.amount.toFixed(2)} ${ingredient.quantity.unit}
               ${specificIngredientName[i]}</li></ul>
               `)
-            }) 
-            recipeObject.instructions.forEach((instruction) => {
-              instructionsSpan.insertAdjacentHTML(
-                "beforebegin",
-                `<li>
+          })
```
Comment (DrewBradley) [inline src/scripts.js:190]: Let's keep an eye out for merge conflicts.

**Review by colefiscus — APPROVED** [review]
So much code, so little time. Looks good to me though!

File: src/styles.scss [hunk 6]
```diff
@@ -70,45 +63,10 @@ body {
 
 main {
   height: 90vh;
-  width: 90vw;
   margin: auto;
+  width: 90vw;
 }
 
-// .search-bar {
```
Comment (DrewBradley) [inline src/styles.scss:77]: Thanks for catching all the commented code!

**Review by DrewBradley — APPROVED** [review]
This CSS and Sass looks so much cleaner!

**Pull request merged**